app/overrides/spree/admin/variants/_form/add_unit_value_and_description.html.haml.deface

```diff
@@ -1,10 +1,13 @@
 / insert_top "[data-hook='admin_variant_form_fields']"
 
 - if product_has_variant_unit_option_type?(@product)
-  .field{"data-hook" => "unit_value"}
-    = f.label :unit_value, "Unit Value"
-    = f.text_field :unit_value, class: "fullwidth"
+  - if @product.variant_unit != 'items'
+    .field{"data-hook" => "unit_value", 'ng-controller' => 'variantUnitsCtrl', 'ng-init' => 'setInitialValues()'}
```

Comment:
This is not really what `ng-init` is for. If you want to run something on initialisation, just run it directly in the controller.